src/_data/chipsets.yml

```diff
@@ -0,0 +1,30 @@
+  arm32: '<span class="material-symbols" style="color: #D43324" aria-label="The Flutter SDK does not support the Arm32 architecture on Linux" role="img">dangerous</span>'
+  arm64: '<span class="material-symbols" style="color: #158477" aria-label="The Flutter SDK supports the Arm64 architecture on Linux" role="img">verified</span> '
+  risc: '<span class="material-symbols" style="color: #D43324" aria-label="The Flutter SDK does not support the RV64GC architecture on Linux" role="img">dangerous</span>'
+- platform: 'macOS'
+  x86: '<span class="material-symbols" style="color: #D43324" aria-label="The Flutter SDK does not support the x86 architecture on macOS" role="img">dangerous</span>'
+  x64: '<span class="material-symbols" style="color: #158477" aria-label="The Flutter SDK supports the x64 architecture on macOS" role="img">verified</span>'
+  arm32: '<span class="material-symbols" style="color: #DADCE0" aria-label="No version of the Flutter SDK exists for the Arm32 architecture on macOS" role="img">do_not_disturb_on</span>'
+  arm64: '<span class="material-symbols" style="color: #158477" aria-label="The Flutter SDK supports the Arm64 architecture on macOS" role="img">verified</span> '
+  risc: '<span class="material-symbols" style="color: #DADCE0" aria-label="No version of the Flutter SDK exists for the RV64GC architecture on macOS" role="img">do_not_disturb_on</span>'
+- platform: 'iOS'
+  x86: '<span class="material-symbols" style="color: #DADCE0" aria-label="No version of the Flutter SDK exists for the x86 architecture on iOS" role="img">do_not_disturb_on</span>'
+  x64: '<span class="material-symbols" style="color: #DADCE0" aria-label="No version of the Flutter SDK exists for the x64 architecture on iOS" role="img">do_not_disturb_on</span>'
+  arm32: '<span class="material-symbols" style="color: #DADCE0" aria-label="No version of the Flutter SDK exists for the Arm32 architecture on iOS" role="img">do_not_disturb_on</span>'
+  arm64: '<span class="material-symbols" style="color: #158477" aria-label="The Flutter SDK supports the Arm64 architecture on iOS" role="img">verified</span>'
+  risc: '<span class="material-symbols" style="color: #DADCE0" aria-label="No version of the Flutter SDK exists for the RV64GC architecture on iOS" role="img">do_not_disturb_on</span>'
+- platform: 'Android'
+  x86: '<span class="material-symbols" style="color: #DADCE0" aria-label="No version of the Flutter SDK exists for the x86 architecture on Android" role="img">do_not_disturb_on</span>'
+  x64: '<span class="material-symbols" style="color: #DADCE0" aria-label="No version of the Flutter SDK exists for the x64 architecture on Android" role="img">do_not_disturb_on</span>'
+  arm32: '<span class="material-symbols" style="color: #DADCE0" aria-label="No version of the Flutter SDK exists for the Arm32 architecture on Android" role="img">do_not_disturb_on</span>'
```

Comment:
This is definitely not correct. 32-bit ARM Android is 100% supported and we have the devices in our lab.

Reply:
We can do another PR. Is there a definitive lust, @zanderso?

Reply:
I would suggest adding the TLs from the teams that support each platform for the definitive answers.